### ISSUE TITLE
Gulp 4 compatibility

### ISSUE THREAD

**Issue by marcandre** [stevelacy/gulp-git#198]
It looks like `gulp-git` is compatible with gulp 4, but the `README` is out of date.

1) It states "Gulp Version 3.x"

2) Many tasks, as written, aren't compatible with gulp 4 and would generate an error "The following tasks did not complete: ....; Did you forget to signal async completion?"

Example:
```
gulp.task('init', function(){
  git.init(function (err) {
    if (err) throw err;
  });
});
```

Could be tweaked like this to work:
```
gulp.task('init', function(done){
  git.init(function (err) {
    if (err) throw err;
    done();
  });
});
```

This definitely would be nicer with async functions.

Let me know if I can help.

**Comment by stevelacy** [comment]
Yes this is compatible with gulp 4 I just don't have the time to update the docs
